src/main/java/com/manardenza/utils/DatabaseSeeder.java

```diff
@@ -21,6 +24,7 @@
     private static final String REIKARTZ = "Reikartz";
     private static final UserDaoImpl USERDAOIMPL = UserDaoImpl.getInstance();
     private static final HotelDaoImpl HOTELDAOIMPL = HotelDaoImpl.getInstance();
+    private static final ReservationDaoImpl RESERVATIONDAOIMPL = ReservationDaoImpl.getInstance();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "RESERVATIONDAOIMPL" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)